slyd/slyd/gitstorage/projects.py

```diff
@@ -138,7 +138,8 @@ def edit_project(self, name, revision):
     @run_in_thread
     def publish_project(self, name, force):
         repoman = self._open_repo(name)
-        if repoman.publish_branch(self._get_branch(repoman), force):
+        if (repoman.publish_branch(self._get_branch(repoman),
+                                   force=force) is True):
```

Comment:
In the past I've had cases where True was not True, when the returned True was from a different module than the tested True. Python only guarantees that None is a singleton so this should be `== True`